src/frontend/frontend/views/story_views.py

```
@@ -96,6 +98,23 @@ class StoryView(BaseView):
+        try:
+            paging_data = PagingData(
+                limit=ASSESS_BOOKMARK_BAR_LIMIT,
+                order="created_asc",
+                query_params={"limit": str(ASSESS_BOOKMARK_BAR_LIMIT), "order": "created_asc"},
+            )
+            return DataPersistenceLayer().get_objects(StoryBookmark, paging_data)
+        except ValidationError as exc:
+            logger.exception(format_pydantic_errors(exc, StoryBookmark))
+        except HTTPException:
+            raise
+        except Exception:
```

Ruff (0.15.17)

[warning] Do not catch blind exception: `Exception`

(BLE001)